requirements/constraints.txt

```diff
@@ -16,4 +16,8 @@ django<2.3
 
 # Requires work, including removal of detail_route. See:
 # https://www.django-rest-framework.org/community/3.8-announcement/#action-decorator-replaces-list_route-and-detail_route
-djangorestframework<3.10.0
\ No newline at end of file
+djangorestframework<3.10.0
+
+# tox-battery 0.6.0 throws when installing dependencies
+# Maybe from https://github.com/signalpillar/tox-battery/pull/24
```

Comment:
There's a proposed fix at https://github.com/signalpillar/tox-battery/pull/26 , linking to that might be more useful.

Reply:
...and it's merged! Let's see if that works.